### DISCUSSION TITLE
storico dati tiene solo 10gg - logbook non trovato

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#3292] [GENERALE - Discussioni generiche]
ciao a tutti, 
mi sono reso conto ora che alcuni sensori (tra cui contatori di energia) hanno lo storico solo di 10gg
ho pensato quindi di rispolverare forse una configurazione che avevo fatto in passato (penso) e di inserire quei determinati sensori nella riga include.
cerco nel file configuration la riga History o recorder ma non esiste.
ora chiedo, è un impostazione di default i 10gg e quindi non avendo nulla dichiarato fa cosi, oppure la sto cercando in un posto errato?

**Comment by Scialla** [comment]
https://www.home-assistant.io/integrations/recorder/
